New snapshot configs (#5817)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/alias.md

This would return the name `analytics.finance.your_snapshot` in the database.

</File>

</VersionBlock>

<VersionBlock firstVersion="1.9">

Configure a snapshots's alias in your `dbt_project.yml` file or config block. 

> For example, if you have a snapshot that represents `your_snapshot` and want to alias it as `updated_at_id`, you would alias like this:

Reviewer: ```suggestion
For example, if you have a snapshot that is named `your_snapshot` and want to alias it as `the_best_snapshot`, you would alias like this:
```

> <File name='dbt_project.yml'>

Reviewer: I think this example shows setting this in a schema yml config file, not the dbt_project.yml

Author: Changed format to the project yaml

> <File name='dbt_project.yml'>

Author: ```suggestion
<File name='schema.yml'>
```

```yml
snapshots:
  - name: your_snapshot
    config:
      database: analytics
      schema: finance
      unique_key: id
      strategy: timestamp
      updated_at: updated_at

>       alias: your_snapshot

Reviewer: ```suggestion
      alias: the_best_snapshot
```

```

> This would return the name `analytics.finance.your_snapshot` in the database.

Reviewer: ```suggestion
This would build your snapshot to `analytics.finance.the_best_snapshot` in the database.
```
